test/TestCliRolePaths.py

```diff
@@ -20,8 +20,6 @@ def run_ansible_lint(self, cwd, bin, role_path, env=None):
             env=env,
         ).communicate()
 
-        self.assertFalse(err, 'Expected no error but was ' + str(err))
```

Comment:
Why? Can this be changed to use a test using a `caplog` fixture?

Reply:
The reason for removing this line is that now we get some valid output on stderr. Presence of output on stderr no longer means an error happened.

At this moment I do not know what kind of test we can reliably do about stderr, it can be empty, or have various content even on perfect valid executions.

Reply:
I noticed that you changed the return code too. So maybe you could assert that? Or how about using `subprocess.check_output()` instead?

Reply:
@webknjaz This method has no knowledge of what is the outcome of the execution and is called 7 times in tests. Doing any change on how we validate the tests should count as outside the scope of implementing the use of python logging.

Reply:
Okay